plugins/flutter_plugins/connectivity_fde/macos/connectivity_fde.podspec

```diff
@@ -0,0 +1,21 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+Pod::Spec.new do |s|
+  s.name             = 'connectivity_fde'
+  s.version          = '0.0.1'
+  s.summary          = 'Flutter plugin for checking connectivity'
+  s.description      = <<-DESC
+  Temporary desktop implmentations of connectivity from flutter/plugins
+                       DESC
+  s.homepage         = 'https://github.com/google/flutter-desktop-embedding/tree/master/plugins/flutter_plugins/connectivity_fde'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'Andrei Lesnitksy' => 'andrei.lesnitsky@gmail.com' }
```

Comment:
sure, wasn't sure about "policy" here, I don't really care about being mentioned as an author since a majority of this plugin was just a rewrite of obj c implementation for ios